test/pack/pack.c

```diff
@@ -228,8 +244,16 @@ void *runtest(void *arg)
         pack_alloc_mem(tbuf_devid, tbufsize, tbuf_memtype, &tbuf_h, &tbuf_d);
 
         yaksa_info_t pack_info, unpack_info;
-        pack_get_ptr_attr(sbuf_d + sobj.DTP_buf_offset, tbuf_d, &pack_info);
-        pack_get_ptr_attr(tbuf_d, dbuf_d + dobj.DTP_buf_offset, &unpack_info);
+        if (sbuf_memtype != MEM_TYPE__DEVICE && tbuf_memtype != MEM_TYPE__DEVICE) {
+            host_only_get_ptr_attr(&pack_info);
+        } else {
+            pack_get_ptr_attr(sbuf_d + sobj.DTP_buf_offset, tbuf_d, &pack_info);
+        }
+        if (tbuf_memtype != MEM_TYPE__DEVICE && dbuf_memtype != MEM_TYPE__DEVICE) {
+            host_only_get_ptr_attr(&unpack_info);
+        } else {
+            pack_get_ptr_attr(tbuf_d, dbuf_d + dobj.DTP_buf_offset, &unpack_info);
+        }
```

Comment:
This patch is forcing all host-only testing to use this new info hint.  This will cause us to lose all testing for the cases where this info hint is not provided.

Reply:
It only adds the info hint to half of the cases, similar to the gpu tests. Refer to line 55 above.